cmd/groundcover/status.go

```diff
@@ -59,29 +59,27 @@ var StatusCmd = &cobra.Command{
 			fmt.Printf("Current groundcover %s is out of date!, The latest version is %s.", release.Version(), chart.Version())
 		}
 
-		if err = waitForAlligators(cmd.Context(), kubeClient, release); err != nil {
+		var nodeList *v1.NodeList
```

Comment:
it looks like the numberOfNodes calc is different here compare to the deploy flow

Reply:
what did we decided about this one?

Reply:
I suggest to left it check if all nodes are monitored, the status cmd requires a product design